src/main/java/org/kaiaccount/account/inter/type/player/AbstractPlayerAccount.java

```diff
@@ -135,6 +136,19 @@ public CompletableFuture<TransactionResult> multipleTransaction(
         return account;
     }
 
+	public boolean deleteBankAccount(@NotNull String name) {
+		Optional<PlayerBankAccount> accountOptional = this.getBank(name);
+		if (accountOptional.isEmpty()) {
+			return false;
```

Comment:
The ``tab`` space seems a bit too large. If your using intellijj then you should be able to click the format button to fix this. 

I personally have ``Format, optimise imports and save`` all on the save key. You can do that with Macros. The example in the Macro setup guide is ``Format and save`` so should give you a good idea if you choose to use it

https://www.jetbrains.com/help/idea/using-macros-in-the-editor.html#reformat_on_save